client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -94,9 +130,32 @@ class Transfer extends React.PureComponent {
 		this.props.paymentSettingsClick( this.props.domain );
 	};
 
+	startTransfer = () => {
+		const { domain, selectedSite, translate } = this.props;
+		this.toggleSubmittingState();
+
+		startInboundTransfer( selectedSite.ID, domain.name, ( error, result ) => {
+			if ( result ) {
+				this.props.successNotice( translate( 'The transfer has been successfully started.' ), {
+					duration: 5000,
+				} );
+				fetchDomains( selectedSite.ID );
+				this.toggleSubmittingState();
+			} else {
+				this.props.errorNotice(
+					error.message || translate( 'We were unable to start the transfer.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We were unable to restart the transfer.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).